lib/ansible/plugins/inventory/kubevirt.py

```diff
@@ -0,0 +1,201 @@
+# Copyright (c) 2018 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    name: kubevirt
+    plugin_type: inventory
+    author:
+      - KubeVirt Team (@kubevirt)
+
+    short_description: KubeVirt inventory source
+
+    description:
+      - Fetch running VirtualMachines for one or more namespaces.
+      - Groups by namespace, namespace_vms  and labels.
+      - Uses kubevirt.(yml|yaml) YAML configuration file to set parameter values.
+
+    options:
+      plugin:
+        description: token that ensures this is a source file for the 'kubevirt' plugin.
+        required: True
+        choices: ['kubevirt']
+      connections:
```

Comment:
missing 'type: list'

Reply:
actually most options are missing 'type'  and none show if there is a default or is not required 
are they all required always?

Reply:
This is actually list of dictionaries. So I can really ell document the values inside the dictionary, because it's not visible in generated documentation.